OpenRA.Mods.Common/Traits/Voiced.cs

```diff
@@ -37,6 +39,7 @@ public Voiced(Actor self, VoicedInfo info)
 		}
 
 		public string VoiceSet { get { return Info.VoiceSet; } }
+		public string SelectVoice { get { return Info.SelectVoice; } }
```

Comment:
...and this to the Selectable trait instead? Then you won't need the prefix, either.